src/addons/editors/widgets/DateRangeFilter.js

```diff
@@ -1110,7 +1111,7 @@ var DateRangeFilter = React.createClass({
 
 	componentDidMount : function(){
 		//initialise jQuery date range widget -
-		var $calendarNode = $(this.refs.calendar.getDOMNode())
+		var $calendarNode = $(ReactDOM.findDOMNode(this.refs.calendar))
```

Comment:
this is wrong. don't need `findDOMNode()` for `refs` anymore. (the refs point to DOM nodes themselves now)